deployments/deployment/nginx-plus-ingress.yaml

```diff
@@ -64,7 +64,6 @@ spec:
         args:
           - -nginx-plus
           - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
-          - -default-server-tls-secret=$(POD_NAMESPACE)/default-server-secret
```

Comment:
do we want to just comment it out as this can still be used with a `secret`? I'm good with either and I'm not sure how we diced what to leave there in a comment 😅